src/scrapers/ams_bgt_scraper.py

```diff
@@ -55,23 +53,55 @@ def parse_buildings(json_response, out_folder='',
     json_response : dict
         JSON response from a WFS request.
     """
-    output_list = []
-
+    parsed_content = []
     for item in json_response['features']:
         pand_id = item['properties']['identificatieBAGPND']
         pand_polygon = item['geometry']['coordinates'][0]
-        x_min, y_max, x_max, y_min = compute_bounding_box(pand_polygon)
 
-        output_list.append([str(pand_id), pand_polygon, x_min, y_max, x_max,
-                           y_min])
+        if prepare_csv:
+            x_min, y_max, x_max, y_min = compute_bounding_box(pand_polygon)
+            parsed_content.append([str(pand_id), pand_polygon, x_min, y_max,
+                                  x_max, y_min])
+        else:
+            parsed_content.append(pand_polygon)
```

Comment:
Waarom geef je hier minder info terug? (geen x_min, x_max etc)
Nu komt het niet overeen met de csv_headers.

Reply:
Zelfde voor regel 96.

Reply:
Ah ok ik snap het, je schrijft nu helemaal geen csv meer weg hier.

Reply:
Klopt! En de min max values gebruik ik om snel in de csv te zoeken. Door alle polygons queryen leek me trager. Moet dit eigenlijk nog testen.

Reply:
TODO time